py/src/ai/h2o/sparkling/ml/algos/H2ODeepLearning.py

```diff
@@ -22,6 +22,8 @@
 from ai.h2o.sparkling.ml.algos.H2OSupervisedAlgoBase import H2OSupervisedAlgoBase
 from ai.h2o.sparkling.ml.params import H2ODeepLearningParams
 
+import sys
```

Comment:
extra import?

Reply:
it's there because of [the line 119](https://github.com/h2oai/sparkling-water/pull/2065/files/1ae375d10918cbef8b5cf5b39abddb9027da083f#diff-fe929c69a7ddb256b7e016988a4b42d4R119).

Reply:
I see, thanks!